presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -854,6 +854,18 @@ This can also be specified on a per-query basis using the ``confidence_based_bro
 Enable treating ``LOW`` confidence, zero estimations as ``UNKNOWN`` during joins. This can also be specified
 on a per-query basis using the ``treat-low-confidence-zero-estimation-as-unknown`` session property.
 
+``optimizer.scalar-function-stats-propagation-enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+* **Type:** ``boolean``
+* **Default value:** ``false``
+
+Enable scalar functions stats propagation via annotations. Annotations `@ScalarFunctionConstantStats` and
+`@ScalarPropagateSourceStats` will define the behavior of the scalar function stats characteristics. This flag or
+the session property `scalar_function_stats_propagation_enabled` when set to true will enable the stats
+propagation via annotations.
```

Comment:
```suggestion
propagation through annotations. This can also be specified on a per-query basis using the ``scalar_function_stats_propagation_enabled`` session property.
```
Avoid using Latin terms and expressions such as via. For discussion and alternate suggestions see the Gitlab documentation style guide [recommended word list entry for via](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#via). 

Also revised the mention of the session property for consistency with the rest of the page.